lambdas/retrieve-cri-credential/src/test/java/uk/gov/di/ipv/core/retrievecricredential/RetrieveCriCredentialHandlerTest.java

```diff
@@ -83,6 +86,17 @@ class RetrieveCriCredentialHandlerTest {
     private static final String TEST_IPV_SESSION_ID = UUID.randomUUID().toString();
     private static final String passportIssuerId = CREDENTIAL_ISSUER_ID;
 
+    private static final String USE_POST_MITIGATIONS_FEATURE_FLAG = "usePostMitigations";
+    private static final SignedJWT TEST_SIGNED_ADDRESS_VC;
+
+    static {
```

Comment:
I think, we can move this static code in below static block.

Reply:
Moved